application-task-date/application-task-date-default/src/main/java/com/xwiki/task/date/internal/DefaultDateConfiguration.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package com.xwiki.task.date.internal;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.inject.Singleton;
+
+import org.xwiki.component.annotation.Component;
+import org.xwiki.configuration.ConfigurationSource;
+import org.xwiki.stability.Unstable;
+
+import com.xwiki.task.date.DateMacroConfiguration;
+
+/**
+ * The default implementation of {@link DateMacroConfiguration}.
+ *
+ * @version $Id$
+ * @since 3.5
```

Comment:
```suggestion
 * @since 3.5.0
```

Since the last version of the task app was 3.4.0 (skipping the bugfixes), it would be nice to keep the consistency.